templates/new-follower-email.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * ActivityPub New Follower E-Mail template.
+ *
+ * @package Activitypub
+ */
+
+if ( ! isset( $actor ) ) {
+	$actor = array();
+	return;
```

Comment:
Could you help me understand why `$actor` gets set to an empty array before returning? Does it has to do with how it's being used in `new_follower()` later on?